hoot-core/src/main/cpp/hoot/core/algorithms/changeset/Change.cpp

```diff
@@ -31,7 +31,8 @@
 namespace hoot
 {
 
-Change::Change()
+Change::Change() :
+_type(Unknown)
```

Comment:
There were a couple of changes where data members weren't given a specific default value.